### PR TITLE
Show GWLF-E runoff results

### PR DESCRIPTION
**Pull request by lewfish**
Run GWLF-E on backend using sample GMS file, and show the runoff results in the UI. 

Before testing, run `scripts/debugcelery.sh` and also run `git pull` in `src/mmw/requirements/src/gwlf-e` to get the latest commits to GWLF-E.

screen shot 2016-04-26 at 12 36 40 pm

Connects #1229 
Connects #1246

### CODE REVIEW

**Comment by kdeloach** [comment]
Nice. This is really cool.

**Comment by kdeloach** [comment]
I ran `pip install -r /opt/app/requirements/base.txt` to install the latest version of gwlfe.

**Comment by kdeloach** [comment]
Consider updating requirements to install gwlf-e from PyPi instead of from source.

**Comment by lewfish** [comment]
> Consider updating requirements to install gwlf-e from PyPi instead of from source.

I made a release of GWLF-E, but then realized it was better for now to just grab it from Github, so it's easy to get the latest updates.

**Comment by kdeloach** [comment]
This looks great. We should update the unit of measurements in the next update.

File: src/mmw/js/src/modeling/gwlfe/runoff/views.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+    templateHelpers: function() {
+        return {
+            runoffVars: runoffVars
+        };
+    },
+
+    modelEvents: {
+        'change:activeVar': 'render'
+    },
```
Comment (kdeloach) [inline src/mmw/js/src/modeling/gwlfe/runoff/views.js]: Consider moving `modelEvents` closer to the `events` hash.

**Comment by kdeloach** [comment]
+1

**Comment by kdeloach** [comment]
One strange thing I observed while testing this is that it looks like gwlf-e is being executed twice. We should investigate this at some point.

File: src/mmw/js/src/modeling/gwlfe/runoff/views.js (hunk 1 above)
Comment (lewfish) [inline src/mmw/js/src/modeling/gwlfe/runoff/views.js]: Done.

**Comment by lewfish** [comment]
> One strange thing I observed while testing this is that it looks like gwlf-e is being executed twice. We should investigate this at some point.

This is because there are two scenarios.

**Comment by kdeloach** [comment]
Oh, I didn't think we created another scenario until the "New Scenario" tab is clicked. Working as designed then.

**Pull request merged**